sentry/sentry.conf.example.py

```diff
@@ -271,6 +278,11 @@ def get_internal_network():
             "organizations:session-replay",
             "organizations:issue-platform",
             "organizations:profiling",
+            "organizations:dashboards-mep",
```

Comment:
Are these orgs designed to test the new snuba containers somehow?

Reply:
those are feature flags that are set at the org level, and utilize the new snuba containers